libs/types/src/enums/announcement-type.enum.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-shadow
+export enum AnnouncementType {
+  Tombstone = 0,
+  Broadcast = 2,
```

Comment:
Actually, this should eventually go away. It's now published in `@dsnp/schemas` here:
https://github.com/LibertyDSNP/dsnp-schemas/blob/main/announcements.ts

Reply:
At the time this was written, that wasn't the case. It seems to map to mainnet schema IDs.
This might be a point of fragility in our content services going forward, if new versions of schemas are published. I'm not as well-versed in the internals of `content-publishing` as you and Puneet.